alcs-frontend/src/app/features/application/review/decision-meeting/decision-meeting.component.ts

```diff
@@ -0,0 +1,70 @@
+import { Component, Input, OnInit } from '@angular/core';
+import { MatDialog } from '@angular/material/dialog';
+import { ApplicationDecisionMeetingDto } from '../../../../services/application/application-decision-meeting/application-decision-meeting.dto';
+import { ApplicationDecisionMeetingService } from '../../../../services/application/application-decision-meeting/application-decision-meeting.service';
+import { ConfirmationDialogService } from '../../../../shared/confirmation-dialog/confirmation-dialog.service';
+import { DecisionMeetingDialogComponent } from '../decision-meeting-dialog/decision-meeting-dialog.component';
+
+@Component({
+  selector: 'app-decision-meeting',
+  templateUrl: './decision-meeting.component.html',
+  styleUrls: ['./decision-meeting.component.scss'],
+})
+export class DecisionMeetingComponent implements OnInit {
+  displayedColumns: string[] = ['date', 'action'];
+  decisionMeetings: ApplicationDecisionMeetingDto[] = [];
+
+  @Input()
+  fileNumber: string = '';
+
+  constructor(
+    public dialog: MatDialog,
+    public decisionMeetingService: ApplicationDecisionMeetingService,
+    private confirmationDialogService: ConfirmationDialogService
+  ) {}
+
+  ngOnInit(): void {
+    this.decisionMeetingService.fetch(this.fileNumber);
+    this.decisionMeetingService.$decisionMeetings.subscribe(
+      (meetings) => (this.decisionMeetings = meetings.sort((a, b) => (a.date >= b.date ? -1 : 1)))
+    );
+  }
+
+  async onCreate() {
+    this.dialog.open(DecisionMeetingDialogComponent, {
+      minWidth: '600px',
+      maxWidth: '800px',
+      width: '70%',
+      data: {
+        fileNumber: this.fileNumber,
+      },
+    });
+  }
+
+  async onEdit(uuid: string) {
+    const meeting = await this.decisionMeetingService.fetchOne(uuid);
```

Comment:
Is this refetch to be careful of others editing the meeting?

Reply:
exactly the reason, otherwise I already have all the data there

Reply:
sounds good